### PR TITLE
Replace old occurrence of configure.in with configure.ac

### PR DESCRIPTION
**Pull request by petk**
The configure.ac is the recommended file to use instead of the old configure.in which will be removed in autotools future versions.

Thank you...

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -20,7 +20,7 @@ config.nice
 config.status
 config.sub
 configure
-configure.in
+configure.ac
```
Comment (TysonAndre) [inline .gitignore]: This is a list of files that should not be tracked in version control. Keep the deleted line.

While configure.ac should indeed not be tracked, configure.in should also not be tracked.

Comment (petk) [reply .gitignore]: I see. I've missed PHP <= 7.1 which still have configure.in. :+1: Ok.

**Comment by petk** [comment]
Added only the new configure.ac instead to .gitignore

**Comment by TysonAndre** [comment]
Thanks!

**Pull request merged**